src/EA0C10.c

```diff
@@ -56,14 +60,187 @@ s32 N(UnkNpcAIFunc26)(Evt* script) {
     return ret;
 }
 
-INCLUDE_ASM(s32, "EA0C10", func_802406CC_EA0FCC); // Similar to UnkFunc7
+// Similar to UnkFunc7
+ApiStatus func_802406CC_EA0FCC(Evt* script, s32 isInitialCall) {
+    DeadEnemy* enemy = (DeadEnemy*) script->owner1.enemy;
+    Npc* npc = get_npc_unsafe(enemy->npcID);
+    Enemy* enemy2;
+    Npc* npc2;
+    f32 posX, posZ;
+    s32 func;
+
+    enemy->unk_108.x = npc->pos.x;
+    enemy->unk_108.y = npc->pos.y;
+    enemy->unk_108.z = npc->pos.z;
+    enemy->unk_114 = 0.0001f;
+    enemy->unk_118 = 0.0001f;
+
+    if (isInitialCall || enemy->unk_B0 & 4) {
+        script->functionTemp[0] = 0;
+        npc->duration = 0;
+        npc->flags |= NPC_FLAG_2 | NPC_FLAG_100;
+        enemy->flags |= ENEMY_FLAGS_100000 | ENEMY_FLAGS_200000 | ENEMY_FLAGS_IGNORE_TOUCH | ENEMY_FLAGS_IGNORE_JUMP | ENEMY_FLAGS_IGNORE_HAMMER | ENEMY_FLAGS_8000000 | ENEMY_FLAGS_10000000;
+        npc->pos.x = 0.0f;
+        npc->pos.y = -1000.0f;
+        npc->pos.z = 0.0f;
+        if (enemy->unk_B0 & 4) {
+            enemy->unk_B0 &= ~4;
+        }
+    }
+
+    func = script->functionTemp[0];
```

Comment:
I don't think this should be called func, since it's not a function. If anything, "temp"